### PR TITLE
run pipelines with telemetry enabled

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: centos.org/pipelines/lib/foremanCentosJob.groovy [hunk 1]
```diff
@@ -53,12 +52,25 @@ pipeline {
                 script {
                     extra_vars = buildExtraVars(extraVars: playBook['extraVars'])
 
+                    def otel_env = """
+                    export OTEL_EXPORTER_OTLP_HEADERS='${env.OTEL_EXPORTER_OTLP_HEADERS}'
+                    export OTEL_EXPORTER_OTLP_ENDPOINT='${env.OTEL_EXPORTER_OTLP_ENDPOINT}'
+                    export OTEL_TRACES_EXPORTER='${env.OTEL_TRACES_EXPORTER}'
+                    export TRACE_ID='${env.TRACE_ID}'
+                    export TRACEPARENT='${env.SPAN_ID}'
```
Comment (evgeni) [inline centos.org/pipelines/lib/foremanCentosJob.groovy:60]: these two don't seem to work as I thought they'd work :(

**Comment by evgeni** [comment]
a run is visible here: https://ui.honeycomb.io/the-foreman/environments/test/result/t3ikdArFcHD/trace/e59tfNwg1SX?fields[]=c_name&fields[]=c_service.name&span=f865ce087d24a96f

**Review by ehelms — APPROVED** [review]
Data capture looks awesome!

**Comment by evgeni** [comment]
One thing that's broken is the tree. The ansible results somehow don't end up as children of the `sh: ssh…` events.

But it's better than nothing, so I'll go ahead and merge it and we can try to fine tune that later

**Pull request merged**